### DISCUSSION TITLE
Bridge v2 offline

### DISCUSSION THREAD

**Discussion by JRBros2346** [bridge-core/editor#270] [Ideas] [answered]
I hereby request to make bridge v2 as offline. Because im an offoine user if bridge and atleast bridge v3 be an offline app

**Comment by LePichu** [comment, marked as answer]
bridge. v2 is also both an online and offline experience, considering you're using a chromium based browser, head to the bridge. v2 website, and click the install icon next to the URL Bar. This will install v2 for offline usage.
